extensions/binding/org.eclipse.smarthome.binding.ntp.test/NtpOSGiTest.launch

```diff
@@ -34,9 +34,9 @@

```

Comment:
This almost doubles the amount of plugins which are started for the test. Are you 100% you need all of them? What about starting from the old config and adding only those which you need additionally?